website/docs/cloud-docs/api-docs/variable-sets.mdx

```diff
@@ -700,3 +700,12 @@ curl \
 [json api document]: /cloud-docs/api-docs#json-api-documents
 
 [json api error object]: https://jsonapi.org/format/#error-objects
+
+
+## Available Related Resources
+
+The GET endpoints above can optionally return related resources, if requested with [the `include` query parameter](/cloud-docs/api-docs/#inclusion-of-related-resources). The following resource types are available:
+
+| Resource Name        | Description                                       |
+| -------------------- | ------------------------------------------------- |
+| `vars`               | Show each variable in a variable set and all of their attributes including  `id`, `key`, `value`, `sensitive`, `category`, `hcl`, `created_at`,  `description` |
```

Comment:
My only other thought is - do we need to list all of the attributes? I wonder if we can just say "Show each variable in a variable set and all of their attributes." I'm kind of back and forth on this, so if you think as an engineer it's more helpful to list them I'm okay with that :) 

```suggestion
| `vars`               | Show each variable in a variable set and all of their attributes including `id`, `key`, `value`, `sensitive`, `category`, `hcl`, `created_at`,  and `description`. |
```